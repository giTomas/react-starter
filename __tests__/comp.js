import React from 'react';
import renderer from 'react-test-renderer';
import App from '../src/app';

describe('app', () => {
  it('renders', () => {
    const component = renderer.create(<App />);
    const json = component.toJSON();
    expect(json).toMatchSnapshot();
  });
});
