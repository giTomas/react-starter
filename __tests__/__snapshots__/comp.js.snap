exports[`app renders 1`] = `
<h1>
  Hello!
</h1>
`;
